### PR TITLE
Add version.h to GPORCA [#105338218]

### PR DESCRIPTION
**Pull request by xinzweb**
Signed-off-by: Jake Frank <jfrank@pivotal.io>

### CODE REVIEW

**Comment by xinzweb** [comment]
@vraghavan78 @jfrank-pivotal

**Comment by vraghavan78** [comment]
Please tag your commit with a message that is informative as to the purpose of the addition.

**Comment by vraghavan78** [comment]
+1

File: libgpopt/version.h.in [hunk 1]
```diff
@@ -0,0 +1,19 @@
+//---------------------------------------------------------------------------
+//	Greenplum Database
+//	Copyright (C) 2015 Pivotal Software, Inc.
```
Comment (danielgustafsson) [inline libgpopt/version.h.in]: 2015?

Comment (xinzweb) [inline libgpopt/version.h.in]: Nice catch, actually, a lot of files have copyright of 2015.

Usually, what do we do with that info? Only for when the file is created? Or, we should bump all the copyright information?

**Pull request merged**